Extension/ui/settings.html

```diff
@@ -648,37 +627,37 @@
         </div>
 
         <div class="section">
-            <div class="section-title">Browse: path</div>
+            <div class="section-title" data-loc-id="browse.path">Browse: path</div>
             <div class="section-text">
-                A list of paths for the Tag Parser to search for headers included by your source files. If omitted, <code>includePath</code> will be used as the <code>path</code>. 
-                Searching on these paths is recursive by default. Specify <code>*</code> to indicate non-recursive search. 
-                For example: <code>/usr/include</code> will search through all subdirectories while <code>/usr/include/*</code> will not.
+                <span data-loc-id="browse.path.description1">A list of paths for the Tag Parser to search for headers included by your source files.</span>

```

Comment:
>browse.path.description1 [](start = 35, length = 24)

What is the process when the descriptions need to be modified? Is it a new description# per sentence?

Reply:
I broke these into separate sentences as a contention that loc should be per sentence.  But, you're right, these should probably be locallized as an entire paragraph so the whole context gets translated, if necessary.  (Though, the contents of the code blocks will not be localized, and will be {0} and {1}, etc., in the string.)

Reply:
As far as process; Any node just needs a "data-loc-id", or any attribute needs a "data-loc-id-<attribute-name>".  If an existing string is modified (with the same id) it may get the original localized strings until the next loc pass, whereas using a new ID will result in the English string until the next loc pass.